### ISSUE TITLE
ill undocumented operates make this repo hard to follow

### ISSUE THREAD

**Issue by tanqi-github** [Aminullah6264/BidirectionalLSTM#2]
there are five matlab files to precess the data. but how this flow from one to a following one?
for example, how to make the label? in the 'onehotLabeling.m' what is the TrainLables variable? how the TotalFeaturesY variable be stored or transited to following operator?
in the python code, how to load the data?...
...

too many ill undocumented operates...make me hard to follow, even i spend a lot of time installing caffe, and to reproduce the amazing results in the paper.
if i miss something, please let me know.

**Comment by aminullah6264** [comment]
I have updated the code to Pytorch, which is much easier to follow.
